specification/sql/resource-manager/Microsoft.Sql/preview/2024-08-01-preview/DatabaseAdvisors.json

```diff
@@ -178,6 +187,32 @@
     }
   },
   "definitions": {
+    "Advisor": {
+      "description": "Database, Server or Elastic Pool Advisor.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
```

Comment:
>v1 [](http://example.com/codeflow?start=35&length=2)

v5

Reply:
Removed code that wasn't related to my change. My change is only related to the additionalProperties of details.